Patch taken from the Desktop consolidation.
See BugDB CR #18372287
Evince garbles japanese characters if ipafont-mincho fonts are installed
for more details.

This patch will be sent upstream if the problem still exists in the latest
version of poppler.

diff -ur poppler-0.14.4.orig/poppler/GfxFont.cc poppler-0.14.4/poppler/GfxFont.cc
--- poppler-0.14.4.orig/poppler/GfxFont.cc	2014年  3月  7日 五 06:08:25
+++ poppler-0.14.4/poppler/GfxFont.cc	2014年  3月  7日 五 06:13:51
@@ -445,7 +445,7 @@
 
 void GfxFont::findExtFontFile() {
   static char *type1Exts[] = { ".pfa", ".pfb", ".ps", "", NULL };
-  static char *ttExts[] = { ".ttf", ".ttc", NULL };
+  static char *ttExts[] = { ".ttf", ".ttc", ".otf", NULL };
 
   if (name) {
     if (type == fontType1) {
diff -ur poppler-0.14.4.orig/poppler/GlobalParams.cc poppler-0.14.4/poppler/GlobalParams.cc
--- poppler-0.14.4.orig/poppler/GlobalParams.cc	2014年  3月  7日 五 06:08:24
+++ poppler-0.14.4/poppler/GlobalParams.cc	2014年  3月  7日 五 06:15:15
@@ -1214,7 +1214,7 @@
         break;
       }
 #endif
-      if (!strncasecmp(ext,".ttf",4) || !strncasecmp(ext, ".ttc", 4))
+      if (!strncasecmp(ext,".ttf",4) || !strncasecmp(ext, ".ttc", 4) || !strncasecmp(ext, ".otf", 4))
       {
         dfp = new DisplayFontParam(fontName->copy(), displayFontTT);  
         dfp->tt.fileName = new GooString((char*)s);
